Leanplum-SDK/Classes/Internal/Leanplum.m

```diff
@@ -51,6 +51,12 @@
 #import "LPOperationQueue.h"
 #include <sys/sysctl.h>
 
+NSString *const kAppKeysFileName = @"LeanplumKeys-Info";
```

Comment:
Rename it to `Leanplum-Info` since we will add additional settings in the future.

Reply:
Thinking about this feature a little bit longer term - it'd be ideal if we could generate and provide this file from the dashboard, instead of them having to make this (it's more inconvenient to create this file than to hard code the api keys)

Given that's the case, should we instead aim for a json file that could work across Android / iOS ?

Reply:
@e7mac That is the next step, as soon as we merge this and android PR we will add support on dashboard to download plist/xml file.

Reply:
We can use json/xml for Android and plist for iOS, downloadable from the Dashboard App Settings Dialog.